src/common/motech-file-upload/motech-file-upload.html

```diff
@@ -0,0 +1,10 @@
+<div class="form-control input-auto input-large fileinput-exists" data-trigger="fileinput">
+    <span class="glyphicon glyphicon-file"></span>&nbsp;
```

Comment:
And also since we are not using glyphicons, I think we have to use font awesome file icon.